test/cron/monthly/host-settlement.test.js

```diff
@@ -157,40 +205,98 @@ describe('cron/monthly/host-settlement', () => {
     expect(attachment).to.have.property('url').that.includes('.csv');
   });
 
   it('should consider fixed fee per host collective', async () => {
-    const reimburseItem = expense.items.find(p => p.description === 'Fixed Fee per Hosted Collective');
+    const reimburseItem = gphHostSettlementExpense.items.find(p => p.description === 'Fixed Fee per Hosted Collective');
     expect(reimburseItem).to.have.property('amount', 100);
   });
 
   it('should update settlementStatus to INVOICED', async () => {
-    const settlements = await models.TransactionSettlement.findAll();
-    expect(settlements.length).to.eq(4); // 1 Platform tip + 3 host fee share
-    settlements.forEach(settlement => {
+    // GBP host
+    const gbpHostTransactions = await models.Transaction.findAll({
+      where: { HostCollectiveId: gbpHost.id },
+      attributes: ['TransactionGroup'],
+      group: ['TransactionGroup'],
+      raw: true,
+    });
+
+    const gbpSettlements = await models.TransactionSettlement.findAll({
+      where: { TransactionGroup: gbpHostTransactions.map(t => t.TransactionGroup) },
+    });
+    expect(gbpSettlements.length).to.eq(4); // 1 Platform tip + 3 host fee share
+    gbpSettlements.forEach(settlement => {
       expect(settlement.status).to.eq(TransactionSettlementStatus.INVOICED);
     });
+
+    // EUR host
+    const eurHostTransactions = await models.Transaction.findAll({
+      where: { HostCollectiveId: eurHost.id },
+      attributes: ['TransactionGroup'],
+      group: ['TransactionGroup'],
+      raw: true,
+    });
+
+    const eurSettlements = await models.TransactionSettlement.findAll({
+      where: { TransactionGroup: eurHostTransactions.map(t => t.TransactionGroup) },
+    });
+    expect(eurSettlements.length).to.eq(2); // Only 1 contribution, but it has platform tip + host fee share
+    eurSettlements.forEach(settlement => {
+      expect(settlement.status).to.eq(TransactionSettlementStatus.INVOICED);
+    });
+  });
+
+  it('settlement should play nicely with taxes', async () => {
+    expect(eurHostSettlementExpense.currency).to.eq('EUR');
+    expect(eurHostSettlementExpense.items).to.have.length(2);
+    expect(eurHostSettlementExpense.items[0].description).to.eq('Platform Tips');
+    expect(eurHostSettlementExpense.items[0].amount).to.eq(300e2);
+    expect(eurHostSettlementExpense.items[1].description).to.eq('Shared Revenue');
+    expect(eurHostSettlementExpense.items[1].amount).to.eq(50e2); // 50€ (100€ host fee * 50% host fee share)
+    expect(eurHostSettlementExpense.amount).to.eq(350e2); // Tips + shared revenue
+  });
+
+  it('collective balance reflects the taxes & host fees properly', async () => {
+    const balanceAmount = await eurCollective.getBalanceAmount();
+    expect(balanceAmount.currency).to.eq('EUR'); // 1000€ contribution - 121€ (100€ host fee + 21€ VAT)
+    expect(balanceAmount.value).to.eq(900e2); // 1000€ contribution - 100€ host fee
```

Comment:
This one is also failing:
Expected: 1000€ - 100€ host fee = 900€
Actual: = 1000€ - 121€ host fee = 879€ (because host fee include tax amount)